src/next-reminders/lib/db.test.ts

```diff
@@ -0,0 +1,171 @@
+import { OneOffSignup } from '../../create-reminder-signup/lambda/models';
+import { createDatabaseConnectionPool, DBConfig } from '../../lib/db';
+import { getNextReminders } from './db';
+
+const config: DBConfig = {
+	url: process.env.TEST_DB_URL ?? '',
+	username: process.env.TEST_DB_USER ?? '',
+	password: process.env.TEST_DB_PASSWORD ?? '',
+};
+
+const pool = createDatabaseConnectionPool(config);
+
+const createOneOffReminder = ({
+	identityId = '0',
+	country = 'GB',
+	reminderPeriod = '2021-01-01',
+	reminderCreatedAt = '2021-01-01',
+	reminderPlatform = 'WEB',
+	reminderComponent = 'EPIC',
+	reminderStage = 'PRE',
+}): OneOffSignup => ({
+	identity_id: identityId,
+	country: country,
+	reminder_period: reminderPeriod,
+	reminder_created_at: reminderCreatedAt,
+	reminder_platform: reminderPlatform,
+	reminder_component: reminderComponent,
+	reminder_stage: reminderStage,
+});
+
+interface SaveOneOffReminderOptions {
+	cancelledAt?: string;
+}
+
+const saveOneOffReminder = (
+	reminder: OneOffSignup,
+	options?: SaveOneOffReminderOptions,
+) => {
+	return pool.query({
+		text: `
+			INSERT INTO one_off_reminder_signups(
+	            identity_id,
+				country,
+	            reminder_period,
+	            reminder_created_at,
+				reminder_cancelled_at,
+	            reminder_platform,
+	            reminder_component,
+	            reminder_stage,
+				reminder_option
+	        ) VALUES (
+	            $1, $2, $3, $4, $5, $6, $7, $8, $9
+	        )
+			`,
+		values: [
+			reminder.identity_id,
+			reminder.country,
+			reminder.reminder_period,
+			reminder.reminder_created_at,
+			options?.cancelledAt,
+			reminder.reminder_platform,
+			reminder.reminder_component,
+			reminder.reminder_stage,
+			reminder.reminder_option,
+		],
+	});
+};
```

Comment:
I'll factor these out into a helpers file

Reply:
👍 presumably this can be shared with the actual lambda code

Reply:
haha yeah we should definitely do that!